assets/javascript/script.js

```diff
@@ -91,4 +91,128 @@ function disappearMobileMenuContact() {
 disAppearMobileMenu.addEventListener('click', disappearMobileMenu);
 disAppearMobileMenuPortfolio.addEventListener('click', disappearMobileMenuPortfolio);
 disAppearMobileMenuAbout.addEventListener('click', disappearMobileMenuAbout);
-disAppearMobileMenuContact.addEventListener('click', disappearMobileMenuContact);
\ No newline at end of file
+disAppearMobileMenuContact.addEventListener('click', disappearMobileMenuContact);
+
+// Dynamic part of website
+
+const works = [
+  {
+    title: 'Tonic',
+    image: 'assets/images/d-snapshot-3.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 'order-0',
+  },
+  {
+    title: 'Multi-post stories',
+    image: 'assets/images/snapshoot-portfolio.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 'order-1',
+  },
+  {
+    title: 'Tonic',
+    image: 'assets/images/snapshot-card-3.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 'order-2',
+  },
+  {
+    title: 'Multi-post stories',
+    image: 'assets/images/snapshot-card-4.svg',
+    see: 'https://nedramrez.github.io/portfolio',
+    repo: 'https://github.com/nedramrez/portfolio',
+    order: 'order-3',
+  },
+];
+
+const details = {
+  client: 'CANOPY',
+  role: 'Back End Dev',
+  counter: 'assets/images/counter.svg',
+  year: '2015',
+  description: 'This represents my expertise in Backend development.',
+};
+
+const tags = {
+  html: 'html',
+  css: 'css',
+  javascript: 'javascript',
+};
+
+// eslint-disable-next-line no-unused-vars
+const detailsPp = function detailsPp() {
+  const x = document.getElementById('pp-details');
+  if (x.style.display === 'block') {
+    x.style.display = 'none';
+  } else {
+    x.style.display = 'block';
+  }
+};
```

Comment:
- [ ] Kindly remove eslint disable commands from your code and try to resolve the errors since linters ensure that you follow best practices and it is a industry standard.
- [ ] Right now, your pop up seems to have static content so no matter which project link I click, it shows up the same pop up with the same content. Kindly ensure that the content of your pop up is also dynamic and the pop up shown depends on which project button you click. That way, you can see the relevant details to that specific project which is how it is supposed to be instead of showing the same popup for every project.